go/tasks/plugins/array/awsbatch/definition/job_def_cache.go

```diff
@@ -27,12 +29,13 @@ type CacheKey interface {
 }
 
 type cacheKey struct {
-	role  string
-	image string
+	role               string
+	image              string
+	jobDefinitionInput batch.RegisterJobDefinitionInput
```

Comment:
hmm this is a process side cache, the cache key could be very large. Is there a simpler way?